src/OrbitGl/ThreadTrack.cpp

```diff
@@ -162,6 +162,15 @@ bool ThreadTrack::IsTrackSelected() const {
   return ToColor(static_cast<uint64_t>(event.color));
 }
 
+Color ThreadTrack::GetTimerColor(const TextBox& text_box, const internal::DrawData& draw_data) {
+  const TimerInfo& timer_info = text_box.GetTimerInfo();
+  uint64_t function_id = timer_info.function_id();
```

Comment:
Nit/Personal Preference: `const` for the variables?

Reply:
I'm a bit on the fence about it, but why not! thanks